__docs__/wonder-blocks-dropdown/combobox.stories.tsx

```diff
@@ -332,7 +332,7 @@ export const ControlledMultilpleCombobox: Story = {
     ],
 
     play: async ({canvasElement}) => {
-        const canvas = within(canvasElement);
+        const canvas = within(canvasElement.ownerDocument.body);
```

Comment:
Interesting, this differs from the examples for these `play()` functions. :(  They always show how you originally had it.

Reply:
Ahh yeah, this is useful for when we are using portals. I should add a comment here stating that.